hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/Federation.md

```diff
@@ -205,9 +205,25 @@ In the same directory we provide scripts to drop the Stored Procedures, the Tabl
 
 **Note:** the FederationStateStoreUser.sql defines a default user/password for the DB that you are **highly encouraged** to set this to a proper strong password.
 
+The versions supported by MySQL are as follows:
+
+1. MySQL 5.7
+2. MySQL 8.0
+MySQL 5.7 and above can be used.
```

Comment:
This text and the note are a little redundant.

Reply:
Thanks for your help reviewing the code, I will modify this part.